JsonToTreeSample/JsonHelper.cs

```diff
@@ -42,7 +42,10 @@ public static TreeViewNode JsonToTree(JContainer obj, string nodeName)
                 {
                     if (token.Type == JTokenType.Object)
                     {
-                        parent.Children.Add(JsonToTree((JObject)token, $"{nodeName}[{index++}]"));
+                        if (nodeName.IndexOf(@"[") != -1)
+                            parent.Children.Add(JsonToTree((JObject)token, $"{nodeName.Substring(0, nodeName.IndexOf(@"["))}[{index++}]"));
```

Comment:
What would be an example string for this case?

Reply:
Actually not so much of an edge case:
{
  "a": "a",
  "b": [{
             "c": "d",
             "e": "f"
         }]
}

I'm not so sure about this commit anymore; there's probably a cleaner way to get the JArray count with each loop. Are you able to skip specific commits in a PR?

Reply:
Specific commits can't be skipped, no. You can make commit changes to your branch, though, and that would update the pr. Or you can close this and set up a new one. 

Also, I think the code already handles that case. Or perhaps were you expecting a different result? 

![image](https://user-images.githubusercontent.com/7649589/50816784-78c52c80-12d6-11e9-9c24-ca27a8b2ec42.png)

Reply:
I'll probably get rid of it, since your local version seems to have a way of showing object or array count already.